server/models/migrations/next/migrate.sql

```diff
@@ -41,3 +41,27 @@ CALL add_column_if_missing('general_ledger', 'principal_center_id', 'MEDIUMINT(8
 
 CALL add_constraint_if_missing('general_ledger', 'general_ledger__cost_center_1', 'FOREIGN KEY (`cost_center_id`) REFERENCES `fee_center` (`id`) ON UPDATE CASCADE');
 CALL add_constraint_if_missing('general_ledger', 'general_ledger__cost_center_2', 'FOREIGN KEY (`principal_center_id`) REFERENCES `fee_center` (`id`) ON UPDATE CASCADE');
+
+/**
+author: @jmcameron
+date: 2021-09-01
+description: adds allocation basis data to cost centers
+*/
+DROP TABLE IF EXISTS `cost_center_basis`;
+CREATE TABLE `cost_center_basis` (
```

Comment:
Removing the `DROP TABLE` instruction, and use the `CREATE TABLE IF NOT EXISTS ` will prevent us from the error below when running the migration script.

![image](https://user-images.githubusercontent.com/5445251/132229474-0d8039b7-d000-4eff-96b7-5542a4d1ed9d.png)

Reply:
I'll push and update.